hasher-matcher-actioner/setup.py

```diff
@@ -4,6 +4,19 @@
 from setuptools import setup
 
 
+extras_require = {
+    "cli": [
+        "pandas",
+        "numpy",
+    ]
+}
+
+all_extras = set(sum(extras_require.values(), []))
+extras_require["test"] = sorted({"pytest"} | all_extras)
+extras_require["package"] = ["wheel"]
+extras_require["lint"] = ["black"]
+extras_require["all"] = sorted(set(sum(extras_require.values(), [])))
```

Comment:
Do we want to install cli dependencies (pandas and numpy) on the lambdas?

Reply:
nvm. I'm assuming we'll need to do `pip install hmalib[all]` for `extras_require['all']`.

So my original question is invalid!

Reply:
exactly, to double check I watched the lambda Docker image build and `pandas` was only included if I added it to `install_requires` below.